File: hw/pastor/u1/hw08FileManagement/chickenInJavaScript/ec.espe.edu.chickenFarmSimulator.model/Egg.js
class Egg {
    constructor() {
        const sizeAvailable = ['S', 'M', 'L'];
        const randomIndex = Math.floor(Math.random() * sizeAvailable.length);
        this.size = sizeAvailable[randomIndex];
    }

    toString() {
        return ` ${this.size}`;
    }

    getSize() {
        return this.size;
    }   

    setSize(size) {
        this.size = size;
    }
}

module.exports = Egg;

File: hw/chalan/u1/hw14Json/Java/src/com/espe/chickenfarm/model/Egg.java
package com.espe.chickenfarm.model;

public class Egg {
    private String size;

    public Egg(String size) {
        this.size = size;
    }

    public String getSize() { return size; }
    public void setSize(String size) { this.size = size; }

    @Override
    public String toString() {
        return String.format("Egg{size=%s}", size);
    }
}